tests/create_packets.py

```diff
@@ -0,0 +1,6 @@
+#! /usr/bin/env python3
```

Comment:
What does this do?

Reply:
If you want to create more test files, run this script and change the payload. I'm going to release a PR after this that is purely comments and documentation